### PR TITLE
New endpoint for returning bids by a bureau cycleposition id

### PR DESCRIPTION
**Pull request by mjoyce91**
Make sure you have the latest mock

Go to Swagger and hit `fsbid/bureau/positions/(some cp_id here)/bids/`

### CODE REVIEW

File: talentmap_api/fsbid/services/bureau.py [hunk 1]
```diff
@@ -67,6 +71,49 @@ def get_bureau_positions_csv(query, jwt_token, host=None, limit=None, includeLim
...
+    return {
+        "emp_id": emp_id,
+        "name": bid.get("full_name"),
+        "grade": bid.get("grade_code"),
+        "skill": f"{bid.get('skill_desc', None)} ({bid.get('skill_code')})",
+        "skill_code": bid.get("skill_code", None),
+        "language": bid.get("language_txt", None),
+        "ted": ted,
+        "has_handshake_offered": hasHandShakeOffered,
+        "submitted_date": ensure_date(bid.get('ubw_submit_dt'), utc_offset=-5),
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/bureau.py:113]: In an effort to learn: How come sometimes we don't pass the default value `None` when using `.get`

File: talentmap_api/settings.py [hunk 2]
```diff
@@ -493,6 +493,7 @@ def config_settings_loader(request):
 
 FSBID_API_URL = get_delineated_environment_variable('FSBID_API_URL', 'http://mock_fsbid:3333')
 EMPLOYEES_API_URL = get_delineated_environment_variable('EMPLOYEES_API_URL', 'http://mock_fsbid:3333/Employees')
+CP_API_URL = get_delineated_environment_variable('CP_API_URL', 'http://localhost:3333/cyclePositions')
```
Comment (elizabeth-jimenez) [inline talentmap_api/settings.py:496]: EP didn't work on my end until I changed this line to `CP_API_URL = get_delineated_environment_variable('CP_API_URL', 'http://mock_fsbid:3333/cyclePositions')
`

File: talentmap_api/fsbid/services/bureau.py (hunk 1 above)
Comment (mjoyce91) [reply talentmap_api/fsbid/services/bureau.py:113]: Good question - I've just been copying this ensure_date function from elsewhere. I actually don't know what `get` defaults to if you don't provide it a fallback value. 🤔

File: talentmap_api/settings.py (hunk 2 above)
Comment (mjoyce91) [reply talentmap_api/settings.py]: Good find, updated

**Pull request merged**